test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/runner/TestRunnerFeature.java

```diff
@@ -1,46 +0,0 @@
-package org.kie.cloud.tests.common.runner;
```

Comment:
@Sgitario Why are you deleting this class?

Reply:
This is caused by a wrong rebase. This class was moved to another module to fix one issue when loading properties from template-xxx.properties.